### PR TITLE
Ready for deployment

### PR DESCRIPTION
**Pull request by mapuya19**
## Summary
- merged MajorForm and SeminarForm into ScheduleGenerator (#12)
- added times and prevent conflicts
- responsive styling (less.js) (#11)
- formatted with Prettier...a lot
- added server files (express.js)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
 const App = () => {
   return (
     <div>
-      <StyledHeader> ALBERTA</StyledHeader>
+      <StyledHeader> ALBERTA </StyledHeader>
       <MainContent>
-        <MajorForm></MajorForm>
-        <SeminarForm />
-        <Majors></Majors>
+        <br />
```
Comment (NicholasLYang) [inline src/App.js:15]: Let's avoid br tags when possible. What if we made the container flexbox?

File: src/App.js [hunk 2]
```diff
@@ -1,28 +1,31 @@
...
+        <br />
+        <center>
+          <h1
+            style={{
```
Comment (NicholasLYang) [inline src/App.js:19]: Any reason we're not using styled components for this?

File: src/Footer.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+const StyledFooter = styled.footer`
+  background-color: "#F8F8F8",
+  border-top: "1px solid #E7E7E7",
```
Comment (NicholasLYang) [inline src/Footer.js:5]: We don't really need the quotes around stuff in here. We could keep them, but then we should convert the other styles accordingly. I don't think that's worth the work, so how about we remove the quotes?

File: src/Table.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+function Table({
+  course1,
```
Comment (NicholasLYang) [inline src/Table.js:4]: This works for now, but we should probably use an array in the future

File: src/constants.js [hunk 5]
```diff
@@ -0,0 +1,2 @@
+export const badMajors = new Set(["COHRT", "FYSEM","MAINT","SCHOL","NODEP"]);
+export const badseminars = new Set(["COHRT", "FYSEM","MAINT","SCHOL","NODEP"]);
```
Comment (NicholasLYang) [inline src/constants.js:2]: seminar should be capitalized

File: src/index.css [hunk 6]
```diff
@@ -1,10 +1,25 @@
+html,
 body {
+  width: 100vw;
+  min-height: 100vh;
+  padding: 0;
   margin: 0;
+  display: flex;
+  /* justify-content: center; */
```
Comment (NicholasLYang) [inline src/index.css:8]: Let's remove instead of commenting

**Comment by mapuya19** [comment]
Made all the changes as requested, with the exception of the Array for outputting the styled table. :)

**Pull request merged**